frontend/src/concepts/k8s/utils.ts

```diff
@@ -5,8 +5,7 @@ import { genRandomChars } from '~/utilities/string';
 export const PreInstalledName = 'Pre-installed';
 
 export const ownedByDSC = (resource: K8sResourceCommon): boolean =>
```

Comment:
probably rename the varible ownedByDSC?

Reply:
yeah, we are renaming this variable in this [PR](https://github.com/opendatahub-io/odh-dashboard/pull/3676/files#diff-c97e72b656ffecb9811bb9488007faa7ddf624b2999454aa361be902c635def8R7).

Reply:
We'll likely want something different than `owned` when we rebase #3676 after this goes in @ppadti

Reply:
yeah sure, I will update it. Thanks